### PR TITLE
Pr json static

### LINKED ISSUE #284: Add versioned third party libraries to the repo (json / queue)
Remove the external library json dependency by adding the latest release to the repo (v3.9.1) enable json comments.
Update the concurrentqueue to release 1.0.3

### PR DESCRIPTION
**Pull request by asedlmayr**
Closes #284 
Adds the json library to the third_party folder
Updates concurrentqueue from 1.02 to 1.03
Enables json comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://falcon.ecg.rice.edu:443/buildStatus/icon?job=github_public_agora%2Fdevelop)](https://falcon.ecg.rice.edu:443/job/github_public_agora/job/develop/)
+[![Build Status](https://falcon.ecg.rice.edu:443/buildStatus/icon?job=github_public_agora%2Fpr-json-static)](https://falcon.ecg.rice.edu:443/job/github_public_agora/job/pr-json-static/)
```
Comment (asedlmayr) [inline README.md:1]:
```suggestion
[![Build Status](https://falcon.ecg.rice.edu:443/buildStatus/icon?job=github_public_agora%2Fdevelop)](https://falcon.ecg.rice.edu:443/job/github_public_agora/job/develop/)
```

File: src/third_party/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+This license file applies to everything in this repository except that which
```
Comment (jianding17) [inline src/third_party/LICENSE.md:1]: Which file is this license used for?

Comment (asedlmayr) [reply src/third_party/LICENSE.md:1]: It is for the concurrentqueue, we can probably take it out or rename it.  We made a decision in RENEWlab to use the json single include file so I have updated this PR also.

Comment (jianding17) [reply src/third_party/LICENSE.md:1]: I think this license is for the whole concurrent queue library. We only take one header file from that repo and it already contains the license on the top of the file. Do we really need an extra license here?

Comment (asedlmayr) [reply src/third_party/LICENSE.md]: I agree, Removed.

**Pull request merged**